src/screens/OtherProfile/index.js

```diff
@@ -628,14 +554,14 @@ const OtherProfile = () => {
           data={handleDataFeed()}
           onScroll={handleScroll}
           onRefresh={handleRefresh}
-          refreshing={loading}
+          refreshing={isLoading}
           onMomentumScrollEnd={(event) => {
             onWillSendViewPostTime(event, feeds);
             fetchNextFeeds(event);
           }}
           ListHeaderComponent={
             <>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The prop `refreshing` of the FlatList component has been updated from `loading` to `isLoading`. Ensure that these two variables have the same meaning in this context.



<!-- This is an auto-generated comment by OSS CodeRabbit -->